staff/NuriaComa/css-layout-simple-adaptive/styles/mobile.css

```diff
@@ -0,0 +1,53 @@
+
+* {
+    padding: 0;
+    margin: 0;
+ }
+
+html, body {
+    height: 100%;
+    min-height: 100% !important;
+    background-color: black;
+    color: white;
+    text-align: center;
+}
+
+.header{
+    width: 100%;
+    height: 20%;
+    border: 1px solid white;
+    box-sizing: border-box;
+}
+
+main{   
+    height: 60%;
+}
+.nav{
+    width: 100%;
+    height: 20%;
+    float: left;
+    border: 1px solid white;
+    box-sizing: border-box;
+}
```

Comment:
Should you use the selector by class when there is not more than one element using that class? investigate the selector by tag. :)

Reply:
DONE! thanks!! :)